.pre-commit-config.yaml

```diff
@@ -32,3 +32,7 @@ repos:
         "--ignore-words",
         ".codespellignore",
       ]
+      exclude: >
```

Comment:
Sorry, I totally missed this in earlier reviews. Why are we excluding all notebooks from Codespell?

If there’s a certain word you would like to use, you can add it to [`.codespellignore`](https://github.com/projectmesa/mesa/blob/main/.codespellignore)

Reply:
Could you please remove all changes to .codespellignore?

Reply:
If i don't add words to .codespellignore the pre-commit check fails on the branch, If I remove them is there any workaround the check?

Reply:
This still needs to be cleaned up

Reply:
I have cleaned it up.

Reply:
Final things done in f4b7eb6.